anyway/telegram_accident_notifications.py

```diff
@@ -131,7 +157,12 @@ def create_public_urls_for_infographics_images(folder_name):
 
 
 def trigger_generate_infographics_and_send_to_telegram(newsflash_id, pre_verification_chat=True):
-    dag_conf = {"news_flash_id": newsflash_id}
-    dag_conf["chat_id"] = TELEGRAM_CHANNEL_CHAT_ID if pre_verification_chat \
-        else TELEGRAM_POST_VERIFICATION_CHANNEL_CHAT_ID
-    trigger_airflow_dag("generate-and-send-infographics-images", dag_conf)
+    infographics_data = get_infographics_data_by_newsflash(newsflash_id)
+    if WIDGETS not in infographics_data:
+        logging.warning(f"no infographics to send for newsflash {newsflash_id}")
+    else:
+        logging.info(f"widgets found in json for {newsflash_id}, triggering dag")
+        dag_conf = {"news_flash_id": newsflash_id}
+        dag_conf["chat_id"] = TELEGRAM_CHANNEL_CHAT_ID if pre_verification_chat \
+            else TELEGRAM_POST_VERIFICATION_CHANNEL_CHAT_ID
+        trigger_airflow_dag("generate-and-send-infographics-images", dag_conf)
```

Comment:
I suggest adding the years_ago param to `dag_conf` to have this flexibility in this dag as well, can be useful in the future